qemu/tests/kvm_unit_test_nested.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import os
+
+from avocado.utils import process
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run nested relstead tests in kvm-unit-test test suite
+
+    1) Clone kvm-unit-tests test suite from repo
+    2) Compile test suite
+    3) Run nested related test cases on host
+    """
+
+    kvm_unit_test_dir = os.path.join(test.logdir, "kvm_unit_tests/")
+    logging.info("kvm_unit_test_dir %s:" % kvm_unit_test_dir)
+    clone_cmd = params["clone_cmd"] % kvm_unit_test_dir
+    process.system(clone_cmd)
+    compile_cmd = params["compile_cmd"] % kvm_unit_test_dir
+    process.system(compile_cmd, shell=True)
+
+    error_context.context("Run kvm_unit_tests on host", logging.info)
+    timeout = params.get_numeric("kvm_unit_test_timeout", 60)
+    run_cmd = params["test_cmd"] % kvm_unit_test_dir
+    logging.info("Run command %s ", run_cmd)
+    status, output = process.getstatusoutput(run_cmd, timeout)
+
+    if output:
```

Comment:
Check `status` wold more meaningful?
`output` always exists.

Reply:
The command filtered failed case and expected failures. So, if no failed test case, there will be no output.

Reply:
Okay, makes sense.